src/utilities/include/public.h

```diff
@@ -31,10 +31,13 @@
 #include <sstream>
 #endif
 
-constexpr std::size_t operator""_z(unsigned long long n) { return n; }
-
 namespace manifold {
 
+/**
+ * Stand-in for C++23's operator""uz (P0330R8)[https://wg21.link/P0330R8].
+ */
+[[nodiscard]] constexpr std::size_t operator""_uz(unsigned long long n) noexcept { return n; }
```

Comment:
This caused me to look up `[[nodiscard]]` - that's a nice language feature, and one we should probably use more. Thanks!

Reply:
Yeah, I'm in the habit of being very pedantic about that and using it nearly everywhere.

Reply:
Improvements like that are always welcome, if you're in the mood to improve our API.

Reply:
Nice to know that this is available with c++17. Thought these attributes are for c++20.